### ISSUE TITLE
[BUG] Build issue against cudf 0.11

### ISSUE THREAD

**Issue by randerzander** [rapidsai/cugraph#566]
I'm using `build.sh libcugraph cugraph` against a source build of cudf 0.11, but I get GDF errors while trying to build cugraph.

It's not clear to me whether Cython bindings need an update, or whether I'm missing a build configuration.

Build failure trace:
```
[18/20] Cythonizing cugraph/utilities/column_utils.pyx                                                            [36/1899]

Error compiling Cython file:
------------------------------------------------------------
...
        category=<void*>category
    )
                                                        
    err = gdf_column_view_augmented(<gdf_column*> &c_col,
                                    <void*> data_ptr,           
                                    <gdf_valid_type*> valid_ptr,
                                    ^
------------------------------------------------------------

cugraph/utilities/column_utils.pyx:59:37: 'gdf_valid_type' is not a type identifier

Error compiling Cython file:
------------------------------------------------------------
...
    )                                                   
                                                     
    err = gdf_column_view_augmented(<gdf_column*> &c_col,       
                                    <void*> data_ptr,       
                                    <gdf_valid_type*> valid_ptr,         
                                    <gdf_size_type> len(col),
                                    ^
------------------------------------------------------------

cugraph/utilities/column_utils.pyx:60:37: 'gdf_size_type' is not a type identifier
                                  
Error compiling Cython file:           
------------------------------------------------------------
...                                                                                                          
    err = gdf_column_view_augmented(<gdf_column*> &c_col,
                                    <void*> data_ptr,                                                            
                                    <gdf_valid_type*> valid_ptr,
                                    <gdf_size_type> len(col),          
                                    gdf_dtype_from_value(col),
                                    <gdf_size_type> col.null_count,
                                    ^
------------------------------------------------------------

cugraph/utilities/column_utils.pyx:62:37: 'gdf_size_type' is not a type identifier

Error compiling Cython file:
------------------------------------------------------------
...
        category=<void*>category
    )

    err = gdf_column_view_augmented(<gdf_column*> c_col,
                                    <void*> data_ptr,
                                    <gdf_valid_type*> valid_ptr,
                                    ^
------------------------------------------------------------

cugraph/utilities/column_utils.pyx:81:37: 'gdf_valid_type' is not a type identifier

Error compiling Cython file:
------------------------------------------------------------
...
    )
    
    err = gdf_column_view_augmented(<gdf_column*> c_col,
                                    <void*> data_ptr,
                                    <gdf_valid_type*> valid_ptr,
                                    <gdf_size_type> col_len,
                                    ^
------------------------------------------------------------

cugraph/utilities/column_utils.pyx:82:37: 'gdf_size_type' is not a type identifier

Error compiling Cython file:
------------------------------------------------------------
...
    err = gdf_column_view_augmented(<gdf_column*> c_col,
                                    <void*> data_ptr,
                                    <gdf_valid_type*> valid_ptr,
                                    <gdf_size_type> col_len,
                                    gdf_dtype_from_value(None, np.int32),
                                    <gdf_size_type> 0,
                                    ^
------------------------------------------------------------

cugraph/utilities/column_utils.pyx:84:37: 'gdf_size_type' is not a type identifier
Traceback (most recent call last):
  File "setup.py", line 63, in <module>
    ext_modules=cythonize(EXTENSIONS),
  File "/conda/envs/rapids/lib/python3.7/site-packages/Cython/Build/Dependencies.py", line 1096, in cythonize
    cythonize_one(*args)
  File "/conda/envs/rapids/lib/python3.7/site-packages/Cython/Build/Dependencies.py", line 1219, in cythonize_one
    raise CompileError(None, pyx_file)
Cython.Compiler.Errors.CompileError: cugraph/utilities/column_utils.pyx
```

**Comment by afender** [comment]
Can't repro. 
Looks like this was addressed by https://github.com/rapidsai/cugraph/pull/564

**Comment by BradReesWork** [comment]
Addressed with PR #564
